src/org/traccar/api/resource/SessionResource.java

```diff
@@ -69,6 +69,12 @@ public User get() throws SQLException {
                     userId = user.getId();
                     request.getSession().setAttribute(USER_ID_KEY, userId);
                 }
+            } else if (request.getParameter("token") != null) {
```

Comment:
Request parameter can be declared as a method argument. See position resource for example.